### DISCUSSION TITLE
Issue with FacadeParser.py File: Missing facadeParserv2.pth Model File

### DISCUSSION THREAD

**Discussion by onurulku23** [NHERI-SimCenter/SimCenterCommon#216] [Brails++]
I am currently working with the FacadeParser.py file, which attempts to load the default facade parser model file from the URL https://zenodo.org/record/10448047/files/facadeParserv2.pth. Unfortunately, it seems that the model file facadeParserv2.pth is not available at the specified location on the Zenodo website, resulting in an error when the code tries to download it.

To resolve this issue, I would appreciate it if you could provide me with the facadeParserv2.pth file. Once I have the file, I can modify the code to load it from a local directory instead of attempting to download it from the non-existent URL.

Thank you very much for your guidance and assistance on this matter. I look forward to your response.

**Comment by JustinBonus** [comment]
There have been a few issues when accessing supplementary files through Zenodo in the past couple months. @bacetiner @fmckenna any idea on what the root of the problem is?

https://github.com/orgs/NHERI-SimCenter/discussions/216
https://github.com/NHERI-SimCenter/BRAILS/issues/213
https://github.com/NHERI-SimCenter/BRAILS/issues/210

**Reply by bacetiner** [reply]
From what I see, there are two different issues here. The URL for the new FacadeParser module was not correctly coded in the source, while the construction type classifier was taken down due to required performance improvements.

**Reply by bacetiner** [reply]
@onurulku23 Thank you for your interest in SimCenter tools. You are correct that the link seems broken, which is why you are observing the issue. I will create another link for the model file and push a new release to address this issue by the end of this week. 

I will send you another message once the new release is up.

**Reply by MujunguMuseru** [reply]
hi there, I am having the same problem, was anyone able to solve this problem?

**Reply by bacetiner** [reply]
Thank you for your interest in SimCenter tools. Could you please confirm whether you encountered this issue while using BRAILS++ or the original BRAILS (which is no longer maintained)?

**Reply by MujunguMuseru** [reply]
i used "pip install git+https://github.com/NHERI-SimCenter/BrailsPlusPlus" to get BRAILS

**Reply by bacetiner** [reply]
It seems you are experiencing this issue using BRAILS++. Could you please share the code you are using for this run so we can rule out possible syntax issues?

**Reply by MujunguMuseru** [reply]
let me clarify something on my end. when i use "pip install brails" i get the url doesnt exist error

then i changed to "pip install git+https://github.com/NHERI-SimCenter/BrailsPlusPlus". Now i cant import the library InventoryGenerator because brails.Inventory does not exist. Has the syntax changed? i am trying to use the example code below provided on the github page.

what i am trying to do is: I have a series of cordinates i want to get property attributes for.

Below is the sample code i am trying to follow.

#import InventoryGenerator:
from brails.InventoryGenerator import InventoryGenerator

# Initialize InventoryGenerator:
invGenerator = InventoryGenerator(location='Berkeley, CA',
                                  fpSource='usastr', 
                                  baselineInv='nsi',
                                  lengthUnit='m',
                                  outputFile='BaselineInvBerkeleyCA.geojson')

# View a list of building attributes that can be obtained using BRAILS:
invGenerator.enabled_attributes()

# Run InventoryGenerator to generate an inventory for the entered location:
# To run InventoryGenerator for all enabled attributes set attributes='all':
invGenerator.generate(attributes=['numstories','roofshape','buildingheight'],
                      GoogleAPIKey='ENTER-YOUR-API-KEY-HERE',
                      nbldgs=100,
                      outputFile='BldgInvBerkeleyCA.geojson')

# View generated inventory:
invGenerator.inventory


when i use "pip install brails" i get the url doesnt exist error

**Reply by bacetiner** [reply]
Yes, BRAILS++ uses a different syntax structure compared to BRAILS. You appear to be using the syntax for BRAILS.

It looks like you are interested in creating a building inventory. Just so I can provide the most relevant BRAILS++ example, could you let me know whether this is for a hurricane or seismic application?

**Reply by MujunguMuseru** [reply]
This is for hurricanes and tornadoes, specifically looking at wind damage. I have about 4000 point coordinates ( Lat + long). I would like to get building attributes at those locations.

**Reply by bacetiner** [reply]
I just have a couple of quick questions before I put together an example for you:
1. Is there a specific set of attributes you are interested in for your study? Please note that we may not yet have full support for all attributes needed to quantify tornado risk.
2. Are you able to share the file containing the point coordinates you mentioned?
